certbot/tests/storage_test.py

```diff
@@ -533,10 +533,9 @@ def test_relevant_values_default(self, mock_parser):
         """Test that relevant_values() can reject a default value."""
         # pylint: disable=protected-access
         from certbot import storage
-        mock_parser.verb = "certonly"
         mock_parser.args = ["--standalone"]
-        mock_action = mock.Mock(dest="rsa_key_size", default=2048)
-        mock_parser.parser._actions = [mock_action]
+        mock_parser.defaults = {"rsa_key_size": 2048}
```

Comment:
What about this? https://github.com/certbot/certbot/blob/f9d5ecaf6fc6776988ddf5ea9dbb010ad45eeb7f/certbot/tests/storage_test.py#L548